docs/getting-started/get-logs-into-loki.md

```diff
@@ -0,0 +1,74 @@
+# Get logs into Loki
+
+After you [install and run Loki](./installation/local.md), you probably want to get logs from other applications into it.
+
+To get application logs into Loki, you need to edit the [Promtail](./clients/promtail/README.md) config file.
+
+Detailed information about configuring Promtail is available in [Promtail configuration](./clients/promtail/configuration.md).
+
+The following instructions should help you get started.
+
+1. If you haven't already, download a Promtail configuration file and put it in the same directory as promtail.exe.
```

Comment:
```suggestion
1. If you haven't already, download a Promtail configuration file and put it in the same directory as promtail binary.
```

Promtail.exe is exclusive to windows, I think here you wanted cover all os ?

Reply:
Correct, thank you!